homeassistant/components/goalfeed/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "goalfeed",
   "name": "Goalfeed",
   "documentation": "https://www.home-assistant.io/integrations/goalfeed",
-  "requirements": ["pysher==1.0.1"],
+  "requirements": ["pysher==1.0.6", "websocket-client==0.57.0"],
```

Comment:
I don't think we should pin websocket-client here in the integration. It will cause conflicts with other integrations. The pysher library should be updated to be compatible with the latest version of websocket-client instead to make it forwards compatible with other libraries that also use websocket-client.

Reply:
Yeah, the repo for that library had a fix merged in in august, but there hasn't been a release cut. I opened an issue to see if they will cut one https://github.com/deepbrook/Pysher/issues/69

Reply:
A release just has been issued:

<https://github.com/deepbrook/Pysher/releases/tag/1.0.7>

Could you take a look @craigjmidwinter ?

Reply:
Thanks @frenck! Tested over night and it looks like it works. Just rebased on dev and updated the PR